bot/client.js

```diff
@@ -15,6 +15,22 @@ client.on('ready', () => {
   util.log('Bot Online and Ready', 0);
 });
 
+// Actions for new guild members
+client.on('guildMemberAdd', (member) => {
+  // Send a the direct welcome message to a guild member
+  member.send(`Welcome ${member} to the Full Sail Armada! Please read our Terms of Service below to get started: \n\n --Terms of Service--`);
+  util.log('Direct Message Sent to New Member', 0);
```

Comment:
Should this be a level 0 debug event?

But good use of including the utility's logging.

Reply:
I believe so, as it is a log message that should appear on the production server, indicating an action that has taken place.